### DISCUSSION TITLE
How to handle Workflows that require manual approval?

### DISCUSSION THREAD

**Discussion by tmljob** [argoproj/argo-workflows#11282] [Q&A] [answered]
For example, a->b, after task a is completed, it is necessary to manually check and confirm the results of task a to confirm whether task b needs to be executed or whether the workflow needs to be ended.

**Comment by zensolution** [comment, marked as answer]
If my understanding is correct, you can refer to this link https://argoproj.github.io/argo-workflows/intermediate-inputs/
